packages/bitcoinml/bitcoinml.0.2.3/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "Davide Gessa <gessadavide@gmail.com>"
+authors: "Davide Gessa <gessadavide@gmail.com>"
+homepage: "https://github.com/dakk/bitcoinml"
+bug-reports: "https://github.com/dakk/bitcoinml/issues"
+license: "MIT"
+dev-repo: "https://github.com/dakk/bitcoinml.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" {build & >= "1.0+beta11"}
+  "base" {build & >= "v0.9.2"}
+  "stdio" {build & >= "v0.9.0"}
+  "configurator" {build & >= "v0.9.1"}
+  "bitstring" {build & >= "2.1.0"}
+  "sexplib" {build & >= "v0.9.2"}
+  "core" {build & >= "v0.9.1"}
+  "ppx_sexp_conv" {build & >= "v0.9.0"}
+  "ppx_jane" {build & >= "v0.9.0"}
+  "ppx_bitstring" {build & >= "2.0.0"}
+  "bignum" {build & >= "v0.9.0"}
+  "cryptokit" {build & >= "1.11"}
+  "stdint" {build & >= "0.3.0-0"}
```

Comment:
I think many of these should not be marked `build`, which indicates that they're not needed at runtime.

And it looks like it might be possible to slim the dependencies a bit -- e.g. removing `core`, which is quite heavyweight and apparently unused.